src/components/list/demoListControls/index.html

```diff
@@ -0,0 +1,29 @@
+<md-list ng-controller="ListCtrl">
+  <md-subheader class="md-no-sticky">Single Action Checkboxes</md-subheader>
+  <md-item ng-repeat="topping in toppings">
+    <p> {{ topping.name }} </p>
+    <md-checkbox class="md-secondary" ng-model="topping.wanted" aria-label="{{ topping.name }}">
```

Comment:
I wonder how many people are going to leave off the `aria-label` here. I think the way you have it coded will already be covered by the `$mdAria` warnings, but I'm wondering if it makes more sense to associate the `<p>` with the checkbox rather than duplicating `{{topping.name}}`. Either with `aria-labelledby` or a more clear element API. Just want to throw it out there though.